src/lustre_dev_tools/cli/start.gleam

```diff
@@ -59,6 +60,14 @@ Watchexec is a popular tool you can use to restart the server when files change.
 
 // STEPS -----------------------------------------------------------------------
 
+fn check_otp_version() -> Cli(Nil) {
+  use <- cli.log("Checking OTP version")
+  case project.otp_version() {
```

Comment:
Is it possible to pull out the int (regex + `int.parse` perhaps) and check `<= 25`? that way we won't break if OTP updates in the future!